docs/upgrade-pro.md

```diff
@@ -0,0 +1,65 @@
+# Upgrade to Percona Server for MySQL Pro
```

Comment:
Please add the list of packages that are available for PRO:
packages deb:
- percona-server-server-pro
- percona-server-test-pro
- percona-server-client-pro
- percona-server-pro-dbg
- percona-server-pro-source
- percona-server-rocksdb-pro
-percona-mysql-router-pro
-[percona-server-pro-common]
packages rpm:
percona-server-server-pro
percona-server-shared-pro
percona-server-client-pro
percona-server-pro-debuginfo
percona-server-rocksdb-pro
percona-server-test-pro
percona-server-devel-pro
percona-mysql-router-pro